l2tscaffolder/scaffolders/plaso.py

```diff
@@ -95,6 +95,25 @@ def _GenerateParserTest(self) -> str:
     return self._mapping_helper.RenderTemplate(
         self.TEMPLATE_PARSER_TEST, self.GetJinjaContext())
 
+  def AddEntriesToInitFiles(self) -> Iterator[Tuple[str, str]]:
```

Comment:
This method is called "AddEntries...." but doesn't seem to actually add anything, just yield some values - rename?